lib/engine/game/g_18_chesapeake_off_the_rails/meta.rb

```diff
@@ -13,9 +13,14 @@ module Meta
         DEPENDS_ON = '18Chesapeake'
 
         GAME_DESIGNER = 'Scott Petersen'
-        GAME_INFO_URL = 'https://github.com/tobymao/18xx/wiki/18Chesapeake'
+        GAME_INFO_URL = 'https://github.com/tobymao/18xx/wiki/18Chesapeake%3A-Off-the-Rails'
         GAME_PUBLISHER = :all_aboard_games
-        GAME_RULES_URL = 'https://www.dropbox.com/s/ivm4jsopnzabhru/18ChesOTR_Rules.png?dl=0'
+        GAME_RULES_URL = {
+          'Base 18Chesapeake Rules' =>
+          'https://cdn.shopify.com/s/files/1/0252/9371/7588/files/18Chesapeake.pdf?v=1597256917',
```

Comment:
This URL is different from the one currently in `g_18_chesapeake/meta.rb`, and it looks like that one still works to me. If that URL ever should change, it would be best if it didn't need to update it in both places:

```suggestion
          G18Chesapeake::Meta::GAME_RULES_URL,
```